cgi-bin/LJ/Talk.pm

```diff
@@ -1491,8 +1491,9 @@ sub talkform {
         default_usertype => 'user',
 
         comment => {
-            editid      => $editid,
-            editreason  => $comment ? $comment->edit_reason : '',
+            editid     => $editid,
+            editreason => $form->{editreason}
+                || ( $comment ? $comment->edit_reason : '' ),
```

Comment:
Should we use `//` instead of `||` here, to allow the user to submit the empty string in order to blank out a previously provided edit_reason? Or does that even work?

Reply:
Oh! Thank you, that's a good catch. (By my understanding, that would work, and I think this is one of the props where the edit_comment function would turn the empty string into a prop removal.) As written, you'd be able to blank out the edit reason on final submit, but we'd revive the old edit reason if you previewed instead, which is inconsistent and not what we want. 

```suggestion
                // ( $comment ? $comment->edit_reason : '' ),
```